### PR TITLE
Boleto refund

### PR DESCRIPTION
**Pull request by henriquekano**

### CODE REVIEW

File: src/main/java/me/pagar/model/RestClient.java [hunk 1]
```diff
@@ -153,7 +153,8 @@ public PagarMeResponse execute() throws PagarMeException {
                     method.equalsIgnoreCase(HttpMethod.DELETE)) {
                 httpClient.setDoOutput(true);
 
-                if (parameters.size() > 1) {
+                if (parameters.size() > 0) {
+                    String json = JSONUtils.getInterpreter().toJson(parameters);
```
Comment (vitorabner) [inline src/main/java/me/pagar/model/RestClient.java]: Is it necessary?

File: src/test/java/me/pagarme/TransactionTest.java [hunk 2]
```diff
@@ -46,7 +51,7 @@ public void testCreatedDateExistence() throws PagarMeException{
     @Test
     public void testCreateAndCaptureTransactionWithOfflineDebitCard() throws Throwable {
 
-        transaction = this.transactionDebitCardCommon("offline");
+        transaction = transactionFactory.create("offline");
```
Comment (vitorabner) [inline src/test/java/me/pagarme/TransactionTest.java]: Change to

```java
transactionFactory.createBoletoTransaction();
transactionFactory.createDebitOfflineTransaction();
transactionFactory.createDebitOnlineTransaction();
```

File: src/test/java/me/pagarme/TransactionTest.java [hunk 3]
```diff
@@ -285,11 +296,32 @@ public void testTransactionAuthAndCaptureRefoundPartialValue() throws Throwable
         Assert.assertEquals(transaction.getRefundedAmount(), PAID_AMOUNT_PARTIAL);
         Assert.assertEquals(transaction.getAuthorizedAmount(), AMOUNT);
     }
+    
+    @Test
+    public void testBoletoTransactionAuthAndCaptureRefund() throws Throwable {
+
+        Transaction transaction = transactionFactory.createBoletoTransaction();
```
Comment (vitorabner) [inline src/test/java/me/pagarme/TransactionTest.java]: Change to
```java
transactionFactory.createBoletoTransaction();
transactionFactory.createDebitOfflineTransaction();
transactionFactory.createDebitOnlineTransaction();
```

**Comment by vitorabner** [comment]
Would you mind changing helper folder name to helpers and factory to factories? Also change TestEndpoints.java to Transaction.java?

**Comment by henriquekano** [comment]
Just noticed it, but all the packages are singular (me.pagar.model, me.pagar.converter, me.pagar.exception). Since they´re all like this, I don´t think changing it now is the right move...

**Comment by vitorabner** [comment]
🚢

**Pull request merged**